cypress/integration/managed_indices_spec.js

```diff
@@ -78,31 +82,46 @@ describe("Managed indices", () => {
     });
   });
 
-  describe.skip("can have policies retried", () => {
+  describe("can have policies retried", () => {
     before(() => {
       cy.deleteAllIndices();
-      // Add a non-existent policy to the index
-      cy.createIndex(SAMPLE_INDEX, POLICY_ID);
-      // Speed up execution time to happen in a few seconds
-      cy.updateManagedIndexConfigStartTime(SAMPLE_INDEX);
+      // Create a policy that rolls over
+      cy.createPolicy(POLICY_ID_ROLLOVER, sampleRolloverPolicy);
+      // Create index with alias to rollover
+      cy.createIndex(SAMPLE_INDEX_ROLLOVER, POLICY_ID_ROLLOVER, { aliases: { "retry-rollover-alias": {} } });
     });
 
     it("successfully", () => {
       // Confirm we have initial policy
-      cy.contains(POLICY_ID);
+      cy.contains(POLICY_ID_ROLLOVER);
+
+      // Speed up execution time to happen in a few seconds
+      cy.updateManagedIndexConfigStartTime(SAMPLE_INDEX_ROLLOVER);
 
       // Wait up to 5 seconds for the managed index to execute
       // eslint-disable-next-line cypress/no-unnecessary-waiting
       cy.wait(5000).reload();
+      cy.get('[data-test-subj="toastCloseButton"]').click({ force: true });
```

Comment:
what toast is this trying to close? this is in several places

Reply:
It's the toast that pops up saying something about not secure cluster, didn't need to close it for this test specifically, but had them always closed for the visual regression tests which will be in follow up PR so the toast isn't included in the pic that's stored.

Reply:
Can attach an image or video clip for this in the next PR, not sure what this actually is 😛